src/store/ViewStore.ts

```diff
@@ -149,17 +149,22 @@ export default class ViewStore {
     removeAll() {
         this.views = [];
         this.updateLocalStorage();
-        logEvent(LogCategories.ChartControl, LogActions.Template, 'Remove All Templates');
+        logEvent(LogCategories.ChartControl, LogActions.Template,'Remove All Templates');
         this.updateRoute('new');
     }
 
-    applyLayout(idx: number, e: TCustomEvent) {
+    applyLayout(idx: number, e: TCustomEvent,onGranularity:(granularity?: TGranularity) => void,onChartType: (chartType?:string) => void) {
```

Comment:
@ahmadtaimoor-deriv it looks like space is missing here before `onGranularity`, before `onChartType`, & before `string`. Could you please run option+shift+f in this file to prettify it ?
Also, i noticed that you're removing `multichart` and `test.tsx` in this PR but you're doing the same in another JS cleanup PR. This will cause a conflict. Why do we need to remove multichart and test.tsx in this PR?